optuna/samplers/_tpe/parzen_estimator.py

```diff
@@ -80,17 +80,17 @@ def _calculate(
                 order: List[int] = []
             else:  # When mus.size is greater than 0.
                 # We decide the place of the  prior.
-                order = numpy.argsort(mus).astype(int)
+                order = numpy.argsort(mus).astype(int).tolist()
```

Comment:
This might be a detail but again, this'll incur an unnecessary copy. How about always making `order` an ndarray instead?